packages/core/elasticsearch/core-elasticsearch-server-internal/src/elasticsearch_config.ts

```diff
@@ -190,6 +190,7 @@ export const configSchema = schema.object({
     { defaultValue: [] }
   ),
   dnsCacheTtl: schema.duration({ defaultValue: 0, min: 0 }),
+  publicBaseUrl: schema.maybe(hostURISchema),
```

Comment:
Is that supposed to be a public configuration setting? If so, it should be documented on the [settings page](https://github.com/elastic/kibana/blob/cc50c8dc943bd904e57c6ee0725c8f2989057b0d/docs/setup/settings.asciidoc)

Reply:
Done in https://github.com/elastic/kibana/pull/191837/commits/bbd297fb5b03feb009e0b41e894a379ffe908823